app/src/main/java/com/pajato/android/gamechat/main/NavigationManager.java

```diff
@@ -132,9 +133,10 @@ public void setAccount(final Account account, final View header) {
         // Determine if there is an account to set up.  Abort if not, otherwise load the account
         // image, display name and email address.
         if (account == null) return;
```

Comment:
If you see these uncaught abuses of standard conditional style, feel free to fix them.